npm/src/filechecks/verify-checks-completed-and-display-banner.ts

```diff
@@ -0,0 +1,35 @@
+import { IFileCheckProgress } from "./get-file-check-progress"
```

Comment:
The name of this file is a combination of the two methods that are in it. What if we add a third method? 🙂 

What about calling it `completed-checks.ts` as a summary? Or something similar - something that's easy to interpret when you see it in the file list.

Reply:
Actually, thinking about this more, would it be better to split this up into two separate files?

It's useful for modules to be at the same level of abstraction - it makes their purpose easier to follow when you read them. Whereas this knows about file check result logic _and_ about updating the DOM.

Reply:
Ok, sure. Yes, I was thinking that the name was too long but I wanted to name it in a way that you instantly knew what it was/did without having to open the file. but yes, splitting it up into smaller files is an even better solution.